src/main/java/com/nerodesk/takes/TsApp.java

```diff
@@ -148,6 +148,7 @@ public void run() {
                     "text/css"
                 )
             ),
+            new FkRegex("/images/.*", new TsClasspath()),
```

Comment:
the image is returned without any Content-Type set. some browsers may complain. let's wrap this into `TsWithType`